make.js

```diff
@@ -367,7 +367,8 @@ target.package = function() {
     // nuspec
     var version = options.version;
     if (!version) {
-        fail('supply version with --version');
+        console.log('ERROR: supply version with --version');
+        process.exit(0);
```

Comment:
can you write something like "Skipping nupkg creation" instead of "ERROR". i want avoid concern over someone seeing "ERROR" in the log and investigating. also, please `return` instead of `process.exit`. process exit is not good from node, stdout is buffered and will be lost sometimes.